src/SFA.DAS.RoatpGateway.Web/Controllers/RoatpGatewayControllerBase.cs

```diff
@@ -83,12 +83,13 @@ public string SetupGatewayPageOptionTexts(SubmitGatewayPageAnswerCommand command
         protected async Task<IActionResult> SubmitGatewayPageAnswer(SubmitGatewayPageAnswerCommand command)
         {
             var username = _contextAccessor.HttpContext.User.UserDisplayName();
```

Comment:
I'm not sure why a Controller has a HttpContextAccessor . I'd personally remove it as it's a code smell